firebase/functions/src/index.ts

```diff
@@ -343,3 +343,63 @@ exports.incProjectProgress = functions.database.ref('/v2/projects/{projectId}/re
 exports.decProjectProgress = functions.database.ref('/v2/projects/{projectId}/requiredResults/').onUpdate(() => {
     return null;
 });
+
+exports.addProjectTopicKey = functions.https.onRequest(async (_, res) => {
+    try {
+        const projectRef = await admin.database().ref('v2/projects').once('value');
+        const data = projectRef.val();
+
+        if (data) {
+            const newProjectData: {[key: string]: string} = {};
+
+            Object.keys(data).forEach((id) => {
+                if (data[id]?.projectTopic) {
+                    const newProjectTopicKey = data[id].projectTopic?.toLowerCase() as string;
```

Comment:
We might need to update this logic to trim whitespace and also replace multiple whitespaces to a single space.